src/Console/AbstractDealwebCommand.php

```diff
@@ -112,25 +117,25 @@ protected function parseVerbosity($level = null)
 
     public function startProcess($message)
     {
-        if (! $this->output->isVerbose()) {
-            $this->output->write(str_pad("   - " . $message, $this->columns - 15, ' ', STR_PAD_RIGHT));
+        if (!$this->output->isVerbose()) {
+            $this->output->write(str_pad('   - '.$message, $this->columns - 15, ' ', STR_PAD_RIGHT));
```

Comment:
Call to method `write` from undeclared class `\Symfony\Component\Console\Output\OutputInterface`